Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17.9 AS deps
+FROM golang:1.18.2 AS deps
```

Comment:
should we lock down everything to 1.18.2 since otherwise it could skew...

Reply:
All the github action scripts don't get auto-updated by dependabot (because the version is a variable to the steps). So every golang patch would be a manual task again, which feels like a step backwards.

I would be more inclined to either have this float (if it's possible) or just assume the skew for patches is tiny as dependabot should promptly make a PR for this file on each golang bump.